ui/qt-lib/src/Ariadne/UI/Qt/MainWindow.hs

```diff
@@ -118,7 +118,7 @@ handleMainWindowEvent langFace putPass = \case
         WalletConfirmationRequest resultVar confirmationType
   UiBackendExceptionEvent (UiBackendException e) -> liftIO $ do
     msg <- QWidget.new
-    void $ QMessageBox.critical msg ("Error" :: String) ("Wallet backend died with exception: " <> show e :: String)
+    runErrorDialog msg ("Wallet backend died with exception: " <> displayException e :: String)
```

Comment:
Maybe
```suggestion
    runErrorDialog msg ("Wallet backend died with an exception: " <> displayException e :: String)
```
?

Reply:
And probably `:: String` shouldn't be needed anymore, because `displayException` is not overloaded.